src/common/Util.ts

```diff
@@ -151,7 +155,7 @@ export default class Util {
         let name = '';
         let extension = '';
         if (nameAndExtensionMatcher.length > 1) {
-            extension = nameAndExtensionMatcher.pop()!.toLowerCase();
+            extension = nameAndExtensionMatcher.pop()!.toLowerCase(); // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
Since there's a `length > 1`, can the `!` non-null assertion operator be removed?

Reply:
Typescript would complain that `Object is possibly 'undefined'`